client/components/board/board-square.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import styled from "styled-components";
+
+const Square = styled.div`
+  border: 1px solid black;
```

Comment:
`background-color: ${props => props.colour}` will avoid needing to hardcode the colour in each of the subclasses.

Reply:
And pass in a constant to supply the colour string, like `<Square colour={colours.grey} />`.